src/games/strategy/triplea/ui/ExtendedStats.java

```diff
@@ -68,11 +68,9 @@ private void fillExtendedStats(final GameData data) {
         m_statsExtended = statsExtended.toArray(new IStat[statsExtended.size()]);
       }
       // add number of techs
-      if (true) {
-        final List<IStat> statsExtended = new ArrayList<>(Arrays.asList(m_statsExtended));
-        statsExtended.add(new TechCountStat());
-        m_statsExtended = statsExtended.toArray(new IStat[statsExtended.size()]);
-      }
+      final List<IStat> statsExtended = new ArrayList<>(Arrays.asList(m_statsExtended));
```

Comment:
Indeed, it is a bit interesting to see how that happened. Fixing this PR has been pretty low on the bucket list. Will get to it one day soon I think..